.github/workflows/_buck-build-test.yml

```diff
@@ -21,11 +21,10 @@ jobs:
           distribution: 'temurin'
 
       - name: Setup miniconda
-        uses: conda-incubator/setup-miniconda@v2
+        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
         with:
-          auto-update-conda: true
           python-version: 3.8
-          activate-environment: build
+          environment-file: .github/requirements/conda-env-ubuntu-X64
 
       - name: Install dependencies
```

Comment:
You would need to remove this step `Install dependencies` to test the change.  The idea is that this step won't be needed anymore

Reply:
Yes, I have removed the block.